static/sass/_pattern_homepage-modal.scss

```diff
@@ -0,0 +1,6 @@
+#homepage-modal {
```

Comment:
I think it might be better to add a `--dark` modifier to the original `p-modal` class than to use an id selector here. There are lots of opinions on this if you search online, eg [this article](https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4), but regardless of personal opinion I think code consistency takes precedence and we don't typically use ids as css selectors